cmd/launcher/main.go

```diff
@@ -34,14 +36,27 @@ func runLaunch() error {
 		return cmd.FailErr(err, "read metadata")
 	}
 
+	var ex launch.ExecFunc
+	if runtime.GOOS == "windows" {
+		ex = func(argv0 string, argv []string, envv []string) error {
+			c := exec.Command(argv0, argv...)
```

Comment:
In this case `argv` will contain `argv0` and you don't want to specify it twice in `exec.Command`

Reply:
Haha, we literally just ran into this 😄  (cc: @micahyoung)

Reply:
Fixed.